docs/2-deploying/1-automated-previews.md

```diff
@@ -41,42 +41,53 @@ jobs:
 
 ### Gitlab CI
 
-This job can be pasted into your `.gitlab-ci.yml` at the root of your repository. You are welcome to change the `stage` to whatever fits your needs to allow you to run tests before the preview is generated, and please be sure to assign correct values for the `variables` in the job. Additionally, you'll need to assign values for `ARCHITECT_EMAIL` and `ARCHITECT_PASSWORD` in your repositories CI variables configuration so that `architect login` will complete successfully.
+This job can be pasted into your `.gitlab-ci.yml` at the root of your repository. You are welcome to change the `stage` to whatever fits your needs to allow you to run tests before the preview is generated, and please be sure to assign correct values for the variables in the job. Additionally, you'll need to assign values for variables in the below config not prefixed with `$CI_` in your repository's CI variables configuration so that the architect commands will run successfully.
+
+This configuration takes advantage of GitLab environments in order to give you better control and visibility into what environments exist and what's deployed to them. On PR creation, both a GitLab and Architect environment will be created. The component specified in the repository will be registered with the Architect Cloud and deployed to the environment. When the PR is either merged or closed, the GitLab environment will be automatically deleted and the component deployed to the environment in the Architect Cloud will be destroyed.
 
 ```yaml
-preview:
-  stage: deploy
+# this example assumes that the repo has ARCHITECT_ACCOUNT and ARCHITECT_PLATFORM set as CI/CD variables
+variables:
+  ARCHITECT_ENVIRONMENT: $CI_MERGE_REQUEST_ID
+  PREVIEW_KEY: preview-$CI_MERGE_REQUEST_ID
+
+stages:
+  - preview
+
+default:
   image: docker:latest
   services:
     - docker:dind
-  only:
-    - merge_requests
-  variables:
-    ARCHITECT_ACCOUNT: my-account
-    PREVIEW_PLATFORM: architect
-    PREVIEW_TAG: $CI_MERGE_REQUEST_SOURCE_BRANCH_NAME
   before_script:
     - apk add --update npm git
     - apk add yq --repository=http://dl-cdn.alpinelinux.org/alpine/edge/community
-    - npm i -g @architect-io/cli --unsafe-perm
+    - npm install -g @architect-io/cli@0.8 --unsafe-perm
```

Comment:
Any reason we decided to version pin here? I get why win pin when we hiding the installation in a github action or CircleCI orb, but since this is under user control it would probably be easier for us to maintain the docs by not pinning here

Reply:
The reason was that the commands could still change in a future version, but I'll remove it and if we update the commands, we can make sure to update the docs

Reply:
Thank you. If we make breaking changes to the integration we should definitely be including docs updates which is why docs live with this repo, but we don't want to be forced to update docs if the flow isn't changing. 

If we had different _versions_ of our docs then I'd agree we should make it fixed, but for not let's be more flexible.